beamer/events.py

```diff
@@ -1,14 +1,14 @@
 import time
 from dataclasses import dataclass
-from typing import Iterable, Optional
+from typing import Iterable, Optional, Union
```

Comment:
`Union` should no longer be needed since we have the new union typing syntax (`A | B`), but even that would not be need were we to use `BaseContract` as mentioned elsewhere.